packages/api/database/migrations/1624831265026_institution_slug_schema.js

```diff
@@ -0,0 +1,47 @@
+/** @type {import('@adonisjs/lucid/src/Schema')} */
+const Schema = use('Schema');
+const Database = use('Database');
+const { createUniqueSlug } = require('../../app/Utils/slugify');
+
+const Institution = use('App/Models/Institution');
+class InstitutionSlugSchema extends Schema {
+	up() {
+		this.alter('institutions', (table) => {
+			// alter table
+			table
+				.string('slug')
+				.after('initials')
+				.unique();
+		});
+
+		this.schedule(async (trx) => {
+			const institutions = await Database.table('institutions').transacting(trx);
+
+			for (const institution of institutions) {
```

Comment:
Testa assim 🙂

```suggestion
			for await (const institution of institutions) {
```

https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Statements/for-await...of

Reply:
Deu certo! Agora a pergunta que não quer calar: A gente deve substituir todos os Promises.all inúteis por isso ai? 🤣